ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -83,7 +93,7 @@ export function PanelHeader({ id, title, description, editHandlers, sx, ...rest
         </InfoTooltip>
       </>
     );
-  } else if (description !== undefined) {
+  } else if (rawDescription !== undefined) {
```

Comment:
I was initially confused why this used `rawDescription` instead of `description`. Guessing it's related to the `null` bits being turned into an empty string. If `useReplaceVariablesInString` returned undefined when the value it received was undefined, that might simplify this a little bit.